rct-demo/app/public/components/common/button.js

```diff
@@ -1,14 +1,14 @@
 import { h } from '/js/src/index.js';
 
-export default function button(model, label, onclickAction, id = '', href = undefined, adClasses='', margin='.m1') {
-    var buttonType = '.myFinalButton' + adClasses + margin;
-    if (model.router) {
+export default function button(model, label, onclickAction, id = '', href = undefined, adClasses='', margin='.m1', forceButton=false) {
```

Comment:
[To jest ogólne zwrócenie uwagi na problem, a nie przytyk do tego, co dopisałeś]: jeżeli nie tworzymy interfejsu, który przyjmuje funkcja `button` (interfejsy są w TypeScripcie, ale nie w JS), to im więcej wrzucamy parametrów, tym mniej czytelny będzie kod w miejscu gdzie korzystamy z buttona (bo parametry nie są podpisane - np. szukając gdzie używasz `forceButton` znajduję tylko plik `button.js`). Trzeba będzie pomyśleć nad jakimś rozwiązaniem, bo przy jeszcze większej liczbie komponentów trudno będzie się znaleźć w kodzie aplikacji.

Reply:
Wiem, że to śrdenio wygąda i włąsnie chciałem to jutro omówić